hazelcast-oss/log4j2.properties

```diff
@@ -4,6 +4,7 @@ appender.console.type=Console
 appender.console.name=STDOUT
 appender.console.layout.type=PatternLayout
 appender.console.layout.pattern=${env:LOGGING_PATTERN}
+appender.console.layout.alwaysWriteExceptions=false
 
-rootLogger.level=${env:LOGGING_LEVEL}
+rootLogger.level=${env:LOGGING_LEVEL:-INFO}
```

Comment:
Is the `-` a typo, or is it a supported syntax?

Reply:
It is the syntax for the default value - see https://logging.apache.org/log4j/2.x/manual/lookups.html.

Reply:
Oh, I see. I thought the syntax was the same as for spring, that is `${LOGGING_LEVEL:INFO}.`